pkg/chains/storage/tekton/tekton.go

```diff
@@ -29,11 +30,12 @@ import (
 )
 
 const (
-	StorageBackendTekton      = "tekton"
-	PayloadAnnotationFormat   = "chains.tekton.dev/payload-%s"
```

Comment:
does removing this annotation result in an "api change"? or are these considered internal implementation details?

Reply:
it could count as an "api change" if people have some client that depends on them (although i don't know of any tbh) -- i think we could play it safe and keep `payload` for now and deprecate it in the next release

we can add this into the [deprecation table](https://github.com/tektoncd/chains/blob/main/docs/deprecations.md#deprecation-table) as well!